main.go

```diff
@@ -31,19 +31,34 @@ type config struct {
 	ManualMerge      bool   `env:"manual_merge,opt[yes,no]"`
 }
 
-func printLogAndExportEnv(gitCmd git.Git, format, env string) error {
+func printLogAndExportEnv(gitCmd git.Git, format, env string, trim bool) error {
 	l, err := output(gitCmd.Log(format))
 	if err != nil {
 		return err
 	}
 
+	if trim {
+		l = checkForTrim(l)
+	}
+
 	log.Printf("=> %s\n   value: %s\n", env, l)
 	if err := exportEnvironmentWithEnvman(env, l); err != nil {
 		return fmt.Errorf("envman export, error: %v", err)
 	}
 	return nil
 }
 
+func checkForTrim(value string) string {
+	const maxLength = 72
```

Comment:
Move the `const maxLength` to global. Usualy we don't like to use global variables, but in later it will be much easier to find and modify the value.

For example:

```
import (
    "fmt"
    ...
)

const 
(
    maxCommitBodyLenght = 72
)

main() {
    ...
    ...
```
}

Reply:
Fixed.